eks/stats.py

```diff
@@ -1,6 +1,105 @@
 import numpy as np
-from sklearn.decomposition import FactorAnalysis
+from sklearn.decomposition import FactorAnalysis, PCA
 from typeguard import typechecked
+from eks.marker_array import MarkerArray, mA_to_stacked_array, stacked_array_to_mA
+from typing import Optional
+
+
+def compute_pca(
+        emA_preds: MarkerArray,
+        emA_vars: MarkerArray,
```

Comment:
at some point maybe we can refactor this to assume that the incoming marker_array contains both predictions and variances? [not high priority]